NorthstarDLL/serverauthentication.cpp

```diff
@@ -114,23 +115,56 @@ void ServerAuthenticationManager::RemovePlayer(R2::CBaseClient* player)
 		m_PlayerAuthenticationData.erase(player);
 }
 
-void ServerAuthenticationManager::VerifyPlayerName(R2::CBaseClient* player, char* authToken, char* name)
+bool checkIsPlayerNameValid(const char* name)
+{
+	int len = strlen(name);
+	if (len >= 64)
```

Comment:
For the `64` we probably should have a macro or constant instead of a :sparkles: magic value :sparkles:

Reply:
I disagree (for now, at least); we do this in so many places for so many things, and it's not safe to change it anyways rn.

Reply:
Moved to #309